packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -2537,6 +2537,24 @@ QUnit.module('Visibility of control elements', {
 });
 
 QUnit.module('Events', moduleConfig, () => {
+    QUnit.test('should output correct size value for panes in onResizeEnd event (T1240125)', function(assert) {
```

Comment:
Isn't it more clear, how do you think? :)
```suggestion
    QUnit.test('items sizes should be already updated when onResizeEnd is raised (T1240125)', function(assert) {
```

Reply:
[85ecf61](https://github.com/DevExpress/DevExtreme/pull/27793/commits/85ecf6149775d598cf2b711ea1ec5f599bb52b25)